### PR TITLE
Original file without any change.

### PR DESCRIPTION
**Pull request by SaravananVijayamuthu**
ADDED
`const message, status , let test_status & let func_message`.

Done success & failure case for `'users/:<id>'`. Added If-else.

Replaced message, status with the above global declaration

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -4,23 +4,42 @@ const app = express();
...
+        name: "Mark"
+    },
+    {
+        id: 2,
+        name: "Jill"
+    },
 ];
+//added message and status
+let function_message = '';
+let test_status = false;
```
Comment (eddiejaoude) [inline server.js]: Camelcase would match the existing style of naming conventions

File: server.js [hunk 2]
```diff
@@ -4,23 +4,42 @@ const app = express();
...
+            success: status,
+            message: message,
+            user: mockUserData.find((user) => user.id == req.params.id)
+        });
+    } else {
+        id = req.params.id;
+        function_message = `Sorry user doesn't exist for the id ${id}`;
+        res.json({
+            success: test_status,
+            message: function_message
```
Comment (eddiejaoude) [inline server.js]: Probably doesn't need a variable as it is used in 1 place only

**Comment by eddiejaoude** [comment]
Thanks for the improvements. I left a a couple of comments

File: server.js (hunk 1 above)
Comment (SaravananVijayamuthu) [reply server.js]: Changed check pls!

File: server.js (hunk 2 above)
Comment (SaravananVijayamuthu) [reply server.js]: Removed it.

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**